### ISSUE TITLE
Compose UI support

### ISSUE THREAD

**Issue by deboer-tim** [containers/podman-desktop#1539]
### Is your feature request related to a problem? Please describe

As we add compose support we need to ensure that it is visible in the UI in the appropriate places.

### Describe the solution you'd like

We do not want to add a top-level nav item for now.

Ideally compose groups (regardless of engine) would show up as groups in the container view similar to pods, with a Details view that shows high-level information.

Some (or maybe all) of this may come along with the core compose support, but we wanted to open this to make sure it is tracked and any remaining work can be targeted to the following sprint.

**Comment by benoitf** [comment]
AFAIK we're already displaying compose groups in the container list

**Comment by deboer-tim** [comment]
Meant to comment a couple weeks ago - yes, we were opening this blind without knowing what was already in the pipe. I'm happy to close as duplicate if this is covered by another issue or once the PR is merged.

**Comment by deboer-tim** [comment]
Compose support is merged, closing old issue.
